WooCommerce/Classes/ViewRelated/Dashboard/StoreStats/StorePerformanceView.swift

```diff
@@ -0,0 +1,117 @@
+import SwiftUI
+
+/// View for store performance on Dashboard screen
+///
+struct StorePerformanceView: View {
+    @ObservedObject private var viewModel: StorePerformanceViewModel
+    @State private var showingCustomRangePicker = false
+
+    init(viewModel: StorePerformanceViewModel) {
+        self.viewModel = viewModel
+    }
+
+    var body: some View {
+        if viewModel.statsVersion == .v4 {
+            VStack(alignment: .leading) {
+                header
+                    .padding(.horizontal, Layout.padding)
+                    .redacted(reason: viewModel.syncingData ? [.placeholder] : [])
+                    .shimmering(active: viewModel.syncingData)
+
+                timeRangeBar
+                    .padding(.horizontal, Layout.padding)
+                    .redacted(reason: viewModel.syncingData ? [.placeholder] : [])
+                    .shimmering(active: viewModel.syncingData)
+
+                Divider()
+
+            }
+            .padding(.vertical, Layout.padding)
+            .background(Color(.listForeground(modal: false)))
+            .clipShape(RoundedRectangle(cornerSize: Layout.cornerSize))
+            .padding(.horizontal, Layout.padding)
+            .sheet(isPresented: $showingCustomRangePicker) {
+                RangedDatePicker(startDate: viewModel.startDateForCustomRange,
+                                 endDate: viewModel.endDateForCustomRange,
+                                 datesFormatter: DatesFormatter(),
+                                 customApplyButtonTitle: viewModel.buttonTitleForCustomRange,
+                                 datesSelected: { start, end in
+                    viewModel.didSelectTimeRange(.custom(from: start, to: end))
+                })
+            }
+        } else {
+            ViewControllerContainer(DeprecatedDashboardStatsViewController())
+        }
+    }
+}
+
+private extension StorePerformanceView {
+    var header: some View {
+        HStack(alignment: .top) {
+            Text(Localization.title)
+                .headlineStyle()
+            Spacer()
+            Menu {
+                Button(Localization.hideCard) {
+                    // TODO
+                }
+            } label: {
+                Image(systemName: "ellipsis")
+                    .foregroundStyle(Color.secondary)
+            }
+        }
+    }
+
+    var timeRangeBar: some View {
+        HStack(alignment: .top) {
+            AdaptiveStack {
+                Text(viewModel.timeRange.tabTitle)
+                    .foregroundStyle(Color(.text))
+                    .subheadlineStyle()
+                Text(viewModel.timeRangeText)
```

Comment:
idea/np: wonder if we can make the left spacing of "Custom Range" be the same size as "Performance" above it:
![Screenshot 2024-04-16 at 20 07 09](https://github.com/woocommerce/woocommerce-ios/assets/266376/29b076f4-e9b8-415d-8e8a-17013675e186)

Reply:
Updated the alignment in d85754c500fcaab843518d8fd195169b47b10b2b.